src/components/editTodoForm/index.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from "react"
+import { useDispatch } from "react-redux"
+import { isEditing, updateTodo } from "redux/actions"
+import ImageButton from "components/buttons/imageButton"
+import { customSanitize } from "utils/sanitizeInput"
+import { DELETE_BUTTON_ALT, IMAGE_PATHS } from "utils/constants"
+import "components/addTodoForm/form.scss"
+
+function EditTodoForm({ todo }) {
+  const dispatch = useDispatch()
+  const [taskTitle, setTaskTile] = useState(todo.text)
+  const [isInputFieldEmpty, setIsInputFieldEmpty] = useState(false)
+
+  function handleEditTodo(event) {
+    event.preventDefault()
+    if (taskTitle.trim() === "") {
+      setTaskTile("")
+      setIsInputFieldEmpty(true)
+    }
+    const satitizedText = customSanitize(taskTitle)
+    if (satitizedText !== "") {
+      dispatch(updateTodo(todo, customSanitize(taskTitle)))
+      setTaskTile("")
+      dispatch(isEditing(false))
+    } else {
+      setTaskTile("")
+      setIsInputFieldEmpty(true)
+    }
+  }
+
+  return (
+    <form className="todo-form__container" onSubmit={handleEditTodo}>
+      <textarea
+        className="todo-form__container__textarea"
+        name="body"
+        placeholder="Add Todo"
+        value={taskTitle}
+        onChange={(e) => setTaskTile(e.target.value)}
```

Comment:
Please try to use normal function instead of arrow functions. and avoid short name.